### PR TITLE
Add medical supplies page

### PR DESCRIPTION
**Pull request by svr333**
Adds the Medical-Supplies page and fixes 1 wrong link

### CODE REVIEW

**Review by Phoenix-Starlight — APPROVED** [review]

**Review by Phoenix-Starlight — APPROVED** [review]
LGTM

File: pages/Medical-Supplies.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+## Table of contents
+* Rag
+* Bandage
+* Splint
+* Vitamins
+* Medicine
+
+### Rag
+Using this item will restore 2 hearts and extinguishes fire if you were to be on fire.
```
Comment (Sfiguz7) [inline pages/Medical-Supplies.md:9]:
```suggestion
Using this item will restore 2 hearts and will extinguish fire if you were to be on fire.
```

File: pages/Medical-Supplies.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+### Bandage
+Using this item will restore 4 hearts and extinguishes fire if you were to be on fire.
```
Comment (Sfiguz7) [inline pages/Medical-Supplies.md:12]:
```suggestion
Using this item will restore 4 hearts and will extinguish fire if you were to be on fire.
```

**Review by Sfiguz7 — CHANGES_REQUESTED** [review]
Since all of the ones below had a double "will" form I think the top two should as well. Other than that LGTM

**Review by Sfiguz7 — APPROVED** [review]
LGTM

**Review by Phoenix-Starlight — APPROVED** [review]
LGTM

**Pull request merged**